### PR TITLE
new status in charm walkthrough

### PR DESCRIPTION
**Pull request by evilnick**
This is just to update the walkthrough to introduce the status-set command

### CODE REVIEW

File: src/en/authors-charm-writing.md [hunk 1]
```diff
@@ -303,9 +303,21 @@ In our script, we are fetching the tarball of the Vanilla software. In these
 cases, it is obviously always better to point to a specific, permanent link to a
 version of the software.
 
-Also, you will notice that we have used the juju-log command. This basically
-spits messages out into the Juju log, which is very useful for testing and
-debugging. We will cover that in more detail later in this walkthrough.
+Also, you will notice that we have used the `juju-log` command and the 
+`status-set` command. These are helper commands for Juju hooks (known as
+"Hook tools") and you will find them covered in more detail
+[in the "how hooks are run" page](authors-hook-environment.html).
```
Comment (petermatulis) [inline src/en/authors-charm-writing.md]: remove "in the" from the visible link?

File: src/en/authors-charm-writing.md [hunk 2]
```diff
@@ -303,9 +303,21 @@ In our script, we are fetching the tarball of the Vanilla software. In these
...
+
+The `status-set` command is used to update the status and message displayed by
+Juju when users run the `juju status` command to see what is going on in the 
+environment. There are a number of pre-defined statuses explained in more 
+detail [on the status reference page](reference-status.html). It is a good idea
```
Comment (petermatulis) [inline src/en/authors-charm-writing.md]: ditto for "on the"?

File: src/en/authors-charm-writing.md [hunk 3]
```diff
@@ -337,15 +349,17 @@ cat <<EOF > $vanilla_config
...
-is a Juju helper command that fetches the named values from the corresponding
-hook on the service we are connecting to. Usually there will be some indication
-of what these values are, but you can always inspect the corresponding hooks to
-find out. In this case we know that when connected, the MySQL charm will create
-a database and generate random values for things like a username and password.
+is another Juju Hook tool, which in this case fetches the named values from 
+the corresponding hook on the service we are connecting to. Usually there will
+be some indication of what these values are, but you can always inspect the
+corresponding hooks to find out. In this case we know that when connected, the 
+MySQL charm will create a database and generate random values for things like a
```
Comment (petermatulis) [inline src/en/authors-charm-writing.md]: Did we agree "mysql" for charms and "MySQL" for services? Or is this the "MySQL service charm? :)

Comment (evilnick) [inline src/en/authors-charm-writing.md]: Aha! Yes, I think this usage is arguably correct. However, I will change it and put it in quotes to make it clearer

File: src/en/authors-charm-writing.md (hunk 2 above)
Comment (evilnick) [inline src/en/authors-charm-writing.md]: ok

**Comment by petermatulis** [comment]
:+1:

**Pull request merged**